{"name": "Invalid Login Test", "status": "passed", "description": "This test verifies that login fails with invalid credentials.", "steps": [{"name": "Navigate to the login page", "status": "passed", "start": 1727261737637, "stop": 1727261739566}, {"name": "Enter invalid credentials", "status": "passed", "start": 1727261739566, "stop": 1727261739608}, {"name": "Attempt to login", "status": "passed", "start": 1727261739608, "stop": 1727261741235}, {"name": "Verify the error message", "status": "passed", "start": 1727261744236, "stop": 1727261744267}], "start": 1727261737637, "stop": 1727261744267, "uuid": "cb3282b9-bc3c-402b-8816-1e918336d1da", "historyId": "2a0d9614e571723b4cc8e0080567c210", "testCaseId": "2a0d9614e571723b4cc8e0080567c210", "fullName": "test_main#test_invalid_login", "labels": [{"name": "suite", "value": "User Authentication"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "Login Tests"}, {"name": "tag", "value": "regression"}, {"name": "host", "value": "DESKTOP-TGQ73LO"}, {"name": "thread", "value": "15788-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}